packages/darklang/stdlib/string.dark

```diff
@@ -118,10 +120,10 @@ module Darklang =
       /// sections](https://www.rfc-editor.org/rfc/rfc4648.html)
       /// [4](https://www.rfc-editor.org/rfc/rfc4648.html#section-4) and
       /// [5](https://www.rfc-editor.org/rfc/rfc4648.html#section-5).
-      let base64Decode
-        (s: String)
-        : PACKAGE.Darklang.Stdlib.Result.Result<String, String> =
-        Builtin.String.base64Decode s
+      let base64Decode (s: String) : PACKAGE.Darklang.Stdlib.Result.Result<String, String> =
+        match (Builtin.Bytes.base64Decode s) with
+        | Ok bytes -> (Builtin.String.fromBytes bytes) |> PACKAGE.Darklang.Stdlib.Result.Result.Ok
```

Comment:
I see here that String.fromBytes ignores errors. If we're putting something in a result I feel we should not do that.

Perhaps we need to add two kinds of String.fromBytes (like we do in Prelude.UTF) so that we don't always just ignore errors.